src/TipBox.vue

```diff
@@ -158,14 +204,37 @@
         height: 22px;
     }
 
+    .close-with-heading {
+        top: auto;
+        padding: 0 1.25rem;
+    }
+
+    .close-with-heading > span {
+        vertical-align: text-top;
+    }
+
     .contents {
         padding: 0 6px;
         width: 100%;
     }
 
+    .transparent-header {
+        background-color: inherit;
+        border-bottom-width: 1px;
+        border-bottom-style: solid;
+        border-bottom-color: inherit;
+        filter: saturate(150%) grayscale(8%);
+    }
+
     .alert-default {
         color: #24292e;
         background-color: #f6f8fa;
         border-color: #e8ebef;
     }
 </style>
+
+<style>
+    div.box-heading > * {
+        margin-bottom: 0;
+    }
+</style>
```

Comment:
Newline

Reply:
Also just curious, is there a reason why we are using a separate `<style>` section?